jsk_apc2016_common/launch/object_segmentation_3d.launch

```diff
@@ -4,16 +4,22 @@
   <arg name="INPUT_IMAGE" />
```

Comment:
this file does not have default value in master branch, too.
default value should be `/camera/rgb/image_raw` or something like that?